monitoring/uss_qualifier/rid/aircraft_state_replayer.py

```diff
@@ -10,53 +10,42 @@
 from monitoring.monitorlib.rid_automated_testing.injection_api import TestFlightDetails, TestFlight, CreateTestParameters, SCOPE_RID_QUALIFIER_INJECT
 from monitoring.monitorlib.typing import ImplicitDict
 import arrow
-import pathlib
 
-from typing import List, Optional
+from typing import List
 from monitoring.uss_qualifier.rid.utils import RIDQualifierTestConfiguration
 
-class TestBuilder():
-    ''' A class to setup the test data and create the objects ready to be submitted to the test harness '''
 
-    def __init__(
-            self, test_configuration: RIDQualifierTestConfiguration,
-            aircraft_state_files: Optional[list] = None) -> None:
-        self.test_configuration = test_configuration
-        # Change directory to read the test_definitions folder appropriately
-        p = pathlib.Path(__file__).parent.absolute()
-        os.chdir(p)
+def get_full_flight_records(aircraft_states_directory: Path) -> List[FullFlightRecord]:
+    """Gets full flight records from the specified directory if they exist"""
 
-        usses = self.test_configuration.injection_targets
+    if not os.path.exists(aircraft_states_directory):
+        raise ValueError('The aircraft states directory does not exist: {}'.format(aircraft_states_directory))
 
-        self.disk_flight_records: List[FullFlightRecord] =[]
-        if not aircraft_state_files:
-            aircraft_states_directory = Path(
-                '../test_definitions/rid', test_configuration.locale, 'aircraft_states')
-            aircraft_state_files = self.get_aircraft_states(aircraft_states_directory)
-
-            for uss_index, uss in enumerate(usses):
-                aircraft_states_path = Path(aircraft_state_files[uss_index])
-                with open(aircraft_states_path, 'rb') as generated_rid_state:
-                    disk_flight_record = ImplicitDict.parse(json.load(generated_rid_state), FullFlightRecord)
-                    self.disk_flight_records.append(disk_flight_record)
-        else:
-            for uss_index, uss in enumerate(usses):
-                generated_rid_state = aircraft_state_files[uss_index]
-                disk_flight_record = ImplicitDict.parse(json.loads(generated_rid_state), FullFlightRecord)
-                self.disk_flight_records.append(disk_flight_record)
+    all_files = os.listdir(aircraft_states_directory)
+    files = [os.path.join(aircraft_states_directory,f) for f in all_files if os.path.isfile(os.path.join(aircraft_states_directory, f))]
 
+    if not files:
+        raise ValueError('The there are no states in the states directory, create states first using the flight_data_generator module.')
```

Comment:
There is a minor typo in the message (`The`). In addition, the new uss_qualifier entrypoint seems to automatically generate the data now. The message could be simplified.

Reply:
Typo resolved, and flight_data_generator reference updated.  It's true that `main` now auto-generates the flight records when they don't exist, but I think the message is still appropriate in the context of this method since the method doesn't know how it's going to be used.